Mathlib/Topology/Separation.lean

```diff
@@ -909,6 +919,17 @@ instance (priority := 100) ConnectedSpace.neBot_nhdsWithin_compl_of_nontrivial_o
     contra (compl_union_self _) (Set.nonempty_compl_of_nontrivial _) (singleton_nonempty _)
   simp [compl_inter_self {x}] at contra
 
+theorem SeparationQuotient.t1Space_iff : T1Space (SeparationQuotient X) ↔ R0Space X := by
+  rw [r0Space_iff, ((t1Space_TFAE (SeparationQuotient X)).out 0 9 :)]
+  constructor
+  · intro h x y xspecy
+    rw [← Inducing.specializes_iff inducing_mk, h xspecy] at *
```

Comment:
Are you sure this `at *` is necessary here? I don’t see how rewriting in an assumption could have any effect here.

Reply:
`rw [← Inducing.specializes_iff] at *` rewrites both the goal `⊢ y ⤳ x` and the assumption `xspecy : x ⤳ y` to `⊢ mk y ⤳ mk x` and `xspecy : mk x ⤳ mk y`.

Then a `rw [h xspecy]` (without `*`) closes the goal by rewriting `⊢ mk y ⤳ mk x` to `⊢ mk y ⤳ mk y`.

Let me know if mathlib style prefers those on separate lines.